src/test/java/com/spaceclub/user/domain/UserTest.java

```diff
@@ -125,4 +134,61 @@ class UserTest {
         );
     }
 
+    @Test
+    void 유저의_상태를_DELETED로_변경에_성공한다() {
+        // given
+        Long userId = 1L;
+        final User user = UserTestFixture.deletedUser(userId, INACTIVE, LocalDateTime.now());
+
+        // when
+        User deletedUser = user.changeStatusToDeleted();
+
+        // then
+        assertAll(
+                () -> assertThat(deletedUser.getStatus()).isEqualTo(DELETED),
+                () -> assertThat(deletedUser.getStatus()).isNotEqualTo(INACTIVE)
+        );
+    }
+
+    @Test
+    void REGISTERED상태의_유저는_유저의_상태를_DELETED로_변경에_실패한다() {
+        // given
+        final User user = UserTestFixture.user1();
+
+        // when, then
+        Assertions.assertThatThrownBy(user::changeStatusToDeleted)
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessage(USER_CANNOT_WITHDRAW.toString());
+    }
+
+    @ParameterizedTest(name = "{index}: 삭제된 시간: {0}")
+    @MethodSource("deletedDateTime")
+    void 유저의_삭제되었는지_상태확인에_성공한다(LocalDateTime deletedAt, boolean expected) {
```

Comment:
메소드 이름을 조금 깔끔하게 바꾸면 좋을 것 같아요
```suggestion
    void 유저의_삭제여부_상태확인에_성공한다(LocalDateTime deletedAt, boolean expected) {
```